swiftUIHW/swiftUIHW/ActivityView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ActivityView.swift
+//  swiftUIHW
+//
+//  Created by Александр Троицкий on 19.01.2023.
+//
+
+import SwiftUI
+
+struct ActivityView: UIViewControllerRepresentable {
+    typealias UIViewControllerType = UIActivityViewController
```

Comment:
```suggestion
    typealias UIViewControllerType = UIActivityViewController
```
Добавь отступы везде посмотри как у ребят срочно